src/dotnet/Properties/AssemblyInfo.cs

```diff
@@ -3,3 +3,4 @@
 
 [assembly: AssemblyMetadataAttribute("Serviceable", "True")]
 [assembly: InternalsVisibleTo("dotnet.Tests")]
+[assembly: InternalsVisibleTo("Microsoft.DotNet.ProjectJsonMigration.Tests")]
```

Comment:
Which test will need InternalsVisibleTo?  The unit tests or the EndToEnd?

I would think the unit tests wouldn't need "dotnet.dll" at all, since it should test the library in isolation.

I would also think the EndToEnd tests actually called `dotnet migrate` from the commandline.  So it wouldn't need any internals.